applications/FluidDynamicsApplication/custom_elements/embedded_fluid_element.cpp

```diff
@@ -122,6 +122,57 @@ void EmbeddedFluidElement<TBaseElement>::CalculateLocalSystem(
     }
 }
 
+template <class TBaseElement>
+void EmbeddedFluidElement<TBaseElement>::Calculate(
```

Comment:
Note that, if you implement one Calculate, you have to declare all its overloads, otherwise we'll get compilation warnings. Feel free to have the overloads just call the base class method.

Reply:
I did not get that compilation warnings. In any case, I'm implementing the overloads as well.

Reply:
That may depend on the compiler/flags you use, but clang at least used to complain about this.